Apps/Playground/Android/app/build.gradle

```
@@ -67,25 +48,18 @@ android {
         }
     }
 
-    externalNativeBuild {
-        cmake {
-            version '3.19.6+'
-            path 'CMakeLists.txt'
-            buildStagingDirectory "../../../../Build/Android"
-        }
-    }
-    packagingOptions {
-        jniLibs {
-            pickFirsts += ['lib/*/libv8android.so', 'lib/*/libjsc.so']
-        }
-    }
-
     namespace 'com.android.babylonnative.playground'
 }
 
+repositories {
+    // Add flatDir repository
+    flatDir {
+        dirs '../../Android/BabylonNative/build/outputs/aar'
```

Isn't this the same as:
```suggestion
        dirs '../BabylonNative/build/outputs/aar'
```